roles/nfd_undeploy/tasks/main.yml

```diff
@@ -18,3 +18,6 @@
 
 - name: Delete the namespace of the NFD Operator
   command: oc --ignore-not-found=true delete -f "{{ nfd_operator_namespace }}"
+
+- name: Delete labels created by the NFD Operator
+  command: oc get no -o yaml | sed -e '/^\s*nfd.node.kubernetes.io/d' -e '/^\s*feature.node.kubernetes.io/d' | kubectl replace -f -
```

Comment:
not a `command` but a `shell`

`no` -> `nodes`

`kubectl` -> `oc`

Reply:
I never use `nodes` jejeje I always use short naming, but ok, I will....